test/python/test_raw_term.py

```diff
@@ -0,0 +1,33 @@
+# RUN: mkdir -p %t
+# RUN: export IN=$(realpath Inputs/raw-term.kore)
+# RUN: cd %t && %kompile "$IN" python --python %py-interpreter --python-output-dir .
+# RUN: KLLVM_DEFINITION=%t %python %s
+
+from test_bindings import kllvm, input_path
+
+import tempfile
+import unittest
+
+
+class TestRawTerms(unittest.TestCase):
+
+    def int_token(self, v):
+        return kllvm.parser.Parser.from_string(f'\\dv{{SortInt{{}}}}("{v}")').pattern()
+
+    def test_raw_terms(self):
+        token = self.int_token(76)
+        term = kllvm.runtime.Term(token)
+
+        with tempfile.NamedTemporaryFile() as fp:
+            term._serialize_raw(fp.name, "SortInt{}")
```

Comment:
What's the primary way to serialize, is it `InternalTerm.serialize` or `Pattern.serialize`?